helm/distributed-idol/values.yaml

```diff
@@ -223,21 +223,41 @@ content:
   #####################################
 
   ##### INTERNAL VARIABLES #####
-  # Values specified below this point in the file should not need to be overriden.
+  # Values specified below this point in the file should not need to be overridden.
   # They are generally here in values.yaml so that DRY principles and a consistent
   # scheme for specifying these values in the templates can be followed.
 
   # -- used to name statefulset, service, ingress
   name: "idol-content"
 
 dah:
+  global:
```

Comment:
Should move to top level global scope

Reply:
Currently required by `idol-library.aciserver.container.base.v1` because I'm passing in `.Values.dah` as the `component` rather than `.Values`.

I've got a TODO to make this `global` section actually global or make it optional in the template though.